hub/core/tensor.py

```diff
@@ -200,6 +256,77 @@ def _check_array_and_tensor_are_compatible(tensor_meta: dict, array: np.ndarray)
         )
 
 
+def read(image_path: str, check_meta: bool = True):
```

Comment:
What is the use of this `read`.

If this is meant to convert image files to np array, then it should be moved out of this class and to other places.